src/tools/tidy/src/main.rs

```diff
@@ -29,6 +29,7 @@ fn main() {
     let concurrency: NonZeroUsize =
         FromStr::from_str(&env::args().nth(4).expect("need concurrency"))
             .expect("concurrency must be a number");
+    let npm: PathBuf = env::args_os().nth(5).expect("need name/path of npm command").into();
```

Comment:
We might want to think about using clap here soon... :laughing:

Reply:
If we're going to refactor how bootstrap passes args to internal tools, I would prefer we use something more type-safe and typo-safe than manually serializing and deserializing args at every process boundary, especially considering these tools aren't really meant to be called directly.

my immediate thought would be putting some types into `build_helper` which get serialized into json blobs that get passed to tools and deserialized. I actually tried doing this in a limited form with compiletest all the way back in #135653, trying to reduce the amount of code that has to be kept in sync on both sides.